README.md

```diff
@@ -49,7 +49,7 @@ In computing an algorithm is the same thing. Just a set of steps for a computer
 program to accomplish a task.
 
 There are often a number of ways your could accomplish a task so learning to
-pick the right algorithm is a key.
+ pick the right algorithm is key.
```

Comment:
"select an _effective_ algorithm is key."

"the right" implies that there is only _one_ "right answer".
when it comes to algorithms, unless we apply a mathematical proof, 
we don't know if something is "ideal", so best describe it as "effective" for current purposes.

> see: "How do we right and wrong?": https://youtu.be/kIIKOXFTeBA

Reply:
Good point. I'll update that in the next commit 👍 
Thanks for the spot 🔍